app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.tsx

```diff
@@ -0,0 +1,38 @@
+import cx from "classnames";
+
+import { DhbVaccineDoseDataForIndexPage } from "../../types/IndexPageProps";
+import dhbDisplayName from "../../utilities/dhbDisplayName";
+
+import { DosesDescriptionList } from "../DosesDescriptionList";
+import { Progress } from "../Progress";
+
+import styles from "./DhbsVaccineDoseDataList.module.scss";
+
+type DhbsVaccineDoseDataListProps = {
+  dhbsVaccineDoseData: DhbVaccineDoseDataForIndexPage[];
```

Comment:
Feels odd with the type naming here, but I think I can tidy this up in my PR next

Reply:
Yeh I would say if we can't pass data through like this it's not a great type name - keen if you can to rename in another pr 🙏